### DISCUSSION TITLE
No Grid Orders class in TailwindCSS v3.4.3 on npm release ?

### DISCUSSION THREAD

**Discussion by FlogFr** [tailwindlabs/tailwindcss#13632] [Help] [answered]
Hello,
I'm working with a HTML / CSS / JS template with tailwindcss v3.4.3 installed from yarn / npm.
I would like to use the Order Grid feature.
In the related TGZ of the release downloaded : https://registry.yarnpkg.com/tailwindcss/-/tailwindcss-3.4.3.tgz#be48f5283df77dfced705451319a5dffb8621519
I don't see any results on grep for `order-last` class.
Could you point me to where am I doing something wrong ?
Thank you

**Comment by wongjn** [comment, marked as answer]
You wouldn't see the `order-last` class name in the package itself. If you use the class name in a file covered by the Tailwind config `content` file globs, the CSS rule should be compiled into your output CSS file.
